### PR TITLE
Documentation for Level 1 Step 2

### PR DESCRIPTION
**Pull request by gabrieledarrigo**
> forEach will hunt me in hell...

### CODE REVIEW

File: docs/level1/02-draw-all-elements-from-response.md [hunk 1]
```diff
@@ -7,3 +7,44 @@ toArray(posts).forEach(function(postToAdd) {
...
+
+### What this code do?
+
+Now We are using the _array_ created with `toArray(posts)` function to draw all the items of the list instead of only one.
+
+> [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)
+
+forEach is a _method_ that can be called on an _array_ and that executes the provided __function__ once for each element of the array.
+
+Paramenters of the provided __function__ are:
```
Comment (Granze) [inline docs/level1/02-draw-all-elements-from-response.md]: typo here :)

File: docs/level1/02-draw-all-elements-from-response.md [hunk 2]
```diff
@@ -7,3 +7,44 @@ toArray(posts).forEach(function(postToAdd) {
...
+For example, We want to log the content of this array with `forEach`:
+
+```js
+var arr = ['a', 'b', 'c', 'd'];
+```
+
+We can write:
+
+```js
+arr.forEach(function(index, value) {
```
Comment (Granze) [inline docs/level1/02-draw-all-elements-from-response.md]: index and value are inverted

Comment (gabrieledarrigo) [inline docs/level1/02-draw-all-elements-from-response.md]: Ops

File: docs/level1/02-draw-all-elements-from-response.md (hunk 1 above)
Comment (gabrieledarrigo) [inline docs/level1/02-draw-all-elements-from-response.md]: Nice catch!

**Pull request merged**